Source/AwsCommonRuntimeKit/mqtt/Mqtt5Packets.swift

```diff
@@ -349,6 +484,34 @@ public class SubackPacket {
         self.reasonString = reasonString
         self.userProperties = userProperties
     }
+
+    static func convertFromNative(_ from: UnsafePointer<aws_mqtt5_packet_suback_view>?) -> SubackPacket? {
```

Comment:
Same for all `convertFromNative` function.

Reply:
Thining about, it is easier to handle the nil case with the static function instead of the init function. With the init function, we need either explicitly check the nil value or handle exception for the nil case.

Reply:
It should be the same. You will need to init?